[16.0][MIG] delivery_mrw: Migration to 16.0

MiguelPoyatos opened the pull request:
Module 'delivery_mrw' migration to 16.0.

ao-landoo committed:
[ADD] l10n_es_ticketbai_oss

Commit:
[UPD] Update l10n_es_ticketbai_oss.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_ticketbai_oss
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_ticketbai_oss/

OCA-git-bot committed:
[UPD] README.rst

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (3 of 3 strings)

Translation: l10n-spain-15.0/l10n-spain-15.0-l10n_es_ticketbai_oss
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-15-0/l10n-spain-15-0-l10n_es_ticketbai_oss/es/

Commit:
[FIX] l10n_es_aeat_mod347: fix added for partner_record's states

EmilioPascual committed:
[IMP] l10n_es_aeat_mod390: Cálculo casillas 85, 95, 97, 98 y 662

@moduon MT-4520

Co-authored-by: Rafael Blasco <8736623+rafaelbn@users.noreply.github.com>
Co-authored-by: Arantxa Sudón <104626993+ArantxaSudon@users.noreply.github.com>
Co-authored-by: Loida Vilaplana <104626797+loida-vm@users.noreply.github.com>
Co-authored-by: Pedro M. Baeza <7165771+pedrobaeza@users.noreply.github.com>

julianram committed:
[ADD] l10n_es_dua_igic : Importaciones con DUAS para Canarias

aritzolea committed:
[FIX] l10n_es_aeat_sii_taxfree: Avoid FacturaExpedida block treatment when cancelling

pedrobaeza commented:
Faltaría el histórico de commits. Revisa por favor https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0

MiguelPoyatos commented:
> Faltaría el histórico de commits. Revisa por favor https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0

Gracias @pedrobaeza

LauraCForgeFlow committed:
[IMP] l10n_es_aeat_mod303_oss: include lines oss in map mod303 2024-10

rlizana reviewed (approved):
LGTM

pedrobaeza commented:
/ocabot migration delivery_mrw

pedrobaeza commented on delivery_mrw/__manifest__.py:
```diff
@@ -3,18 +3,21 @@
+    "external_dependencies": {
+        "python": [
+            "zeep",
+        ],
```
@MiguelPoyatos veo bastante cambios de estos que no añaden valor por haber añadido una coma al final. Por favor deshazlos todos que no metan ruido al commit de migración.

MiguelPoyatos replied:
Ok, los guardo para un PR posterior. Gracias

etobella committed:
[FIX] l10n_es_aeat_mod190: Multiple fixes

Compute for all lines, otherwise, the tax is not correctly assigned.

Segui-S73 committed:
[FIX] l10n_es_aeat_mod349: Solve error where rectifications for other periods do not have the original amount

QuiJoQuim committed:
Translated using Weblate (Catalan)

Currently translated at 96.1% (25 of 26 strings)

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_vat_prorate
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_vat_prorate/ca/

QuiJoQuim committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (170 of 170 strings)

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_facturae
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_facturae/ca/

QuiJoQuim committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (147 of 147 strings)

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod111
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod111/ca/

pedrobaeza committed:
[IMP] l10n_es_vat_prorate: Add hook for VAT prorate move line

This way, you can inject any possible extra value.

OCA-git-bot committed:
[BOT] post-merge updates

victoralmau committed:
[IMP] l10n_es_aeat_sii_oca: Improve SII not sent filter to use sii_enabled field

TT52143

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[FIX] l10n_es_vat_book: Incorrect mapping for non deductible

And the 5% ND was missing.

Fixup of #3813

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[FIX] l10n_es_aeat_mod303: Missing "to regularize" flags

IVA 2% + Recargo equivalencia 0.26% should be included in VAT
regularization entry.

TT52149

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[FIX] l10n_es_aeat_mod303: Fields 14, 15 and 80 miss the new taxes

OCA-git-bot committed:
[BOT] post-merge updates

EmilioPascual committed:
[IMP] Update pre-commit

@moduon

etobella committed:
[IMP] l10n_es_mis_report: Include 4757 and 4707

pedrobaeza committed:
[IMP] l10n_es_vat_prorate: Include new october 2024 taxes

Following odoo/odoo#180812

TT52270

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[IMP]l10n_es_aeat_mod347_igic: propose maintainer

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update l10n_es_aeat_mod390.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod390
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod390/

chienandalu committed:
[FIX] l10n_es_pos: compatibility with kitcheb order printers

As we use the simplified invoice ref, the internal reference is usually
useless. But when a kitchen order is printed that's the only reference
we've got. This way we can show that reference in the ticket only when
kitchen printers are configured in the point of sale.

TT52521

EmilioPascual committed:
[FIX] l10n_es_aeat_mod303: Error generación asiento contable cuando sale a pagar pero se compensa parcialmente con ejercicios anteriores

@moduon MT-8455

OCA-git-bot committed:
[BOT] post-merge updates

chienandalu committed:
[FIX] l10n_es_aeat_sii_oca: cancel one invoice

If we really want to cancel just one invoice we should send the whole
recordset, as the effect will be looping over the same tasks over and
over again.

TT52550

EmilioPascual committed:
[FIX] Cálculo de de retenciones y percepeciones dinerarias/en especie por incapacidad

Con la condición actual
`x.discapacidad or x.discapacidad != "0`
si el contacto no tiene marcada ninguna discapacidad se calcula también como discapacidad.

@moduon

OCA-git-bot committed:
[BOT] post-merge updates

EmilioPascual committed:
[IMP] l10n_es_aeat_sii_oca: Impuesto ganadería y pesca (10,5%)

@moduon MT-8089

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[IMP] l10n_es_aeat: Reverse order for reports

It's more significant to have first the latest AEAT reports than the
first ones.

pedrobaeza committed:
[IMP] l10n_es_aeat_mod303: Follow 2 column layout in exonerated 390

If not, the fields are stretched on regular screens, and they can't be
filled.

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

etobella committed:
[FIX] l10n_es_aeat_mod190: Add modifications for 2024

pedrobaeza committed:
[IMP] l10n_es_aeat_mod390: 2024 tax mapping and BOE export format

Changed according https://sede.agenciatributaria.gob.es/static_files/Sede/Disenyo_registro/DR_300_399/archivos_24/dr390e2024.xlsx

OCA-git-bot committed:
[BOT] post-merge updates

etobella committed:
[REF] l10n_es_aeat_mod190: Compute using group by

etobella committed:
[FIX] l10n_es_aeat_mod190: Add missing taxes

pedrobaeza committed:
[IMP] l10n_es_aeat_mod347: New october 2024 taxes mapping

TT52973

pedrobaeza committed:
[FIX] l10n_es_atc: No auto-install

It should be installed on demand, as having AEAT doesn't mean we should
install ATC.

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[IMP] l10n_es_aeat_mod347: Don't auto-delete 347 mail

This way, we preserve the message on the partner record chatter for
traceability.

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[IMP] l10n_es_vat_book + l10n_es_aeat: Drill through on tax summary

Allow to drill through in the tax summary to list journal items that
leads to the base and tax amounts.

As we are on this, there's a tool method now in base module l10n_es_aeat
for opening this kind of view, and fixing that the journal items was
always open with move grouping by default.

TT52996

pedrobaeza committed:
[FIX] l10n_es_aeat_mod390: Missing 0% taxes in field 99

Check the equivalent one in 303:

https://github.com/OCA/l10n-spain/blob/0634eb5ed07baa266f3436698d21f5127c2c4cb7/l10n_es_aeat_mod303/data/2024-10/l10n.es.aeat.map.tax.line.csv#L47

TT52996

pedrobaeza committed:
[FIX] l10n_es_aeat_mod303: Remove field 96

It was incorrectly mapped to "recargo de equivalencia" base amounts,
which is not correct, as that base amounts are already included in
regular operations, so it appears duplicated.

This field is for being filled in specific cases where the subject is
on the special fiscal regime "Recargo de equivalencia".

TT52996

Commit:
[UPD] Update l10n_es_vat_book.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_vat_book
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_vat_book/

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

rrebollo committed:
[IMP] l10n_es_pos_by_device: remove overwrite of dead code open_session_cb method from pos.config

etobella committed:
[IMP] l10n_es_aeat_mod190: Allow to edit

Commit:
[UPD] Update l10n_es_aeat_mod190.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod190
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod190/

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update l10n_es_aeat_mod303.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod303
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod303/

sergiobstoj committed:
[IMP] l10n_es_aeat_mod347: Display move name or move ref depending on its move_type

aritzolea committed:
[IMP] l10n_es_aeat_mod347: Refactor mail sending code

rrebollo committed:
[IMP] l10n_es_ticketbai: refactor create method to support multi-record creation in account.move

ofonseca-pyming committed:
[16.0][FIX] Remove dependencies already specified in the Odoo requirements
The following dependencies: cryptography, zeep, and requests are used in
different modules, but we remove them from being required because they are
already included in Odoo's requirements.txt.
Having them in both places creates maintenance overhead and potential conflicts.

Commit:
[UPD] Update l10n_es_aeat_mod347.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod347
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod347/

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Take into account inactive taxes

Since odoo/odoo#179797, IVA 5% taxes templates are deactivated, and if
you calculate a 347 report containing them, they are not included.

This commit takes this situation into account.

This is similar to what has been done in #3744 and 3758

TT54641

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Avoid fake quarter amounts trackings

Due to a ORM glitch, when recomputing 347 partner records, there are
tracking messages saying that the quarters amount have changed, which
is not really that way, so let's compare before doing the assignation
for avoiding it if no real amount change.

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[IMP] l10n_es_aeat_sii: Annotate F6 lack

Closes #3941

OCA-git-bot committed:
[UPD] README.rst

LauraCForgeFlow committed:
[FIX] l10n_es_aeat_mod390: Calculation of field 85

Commit:
[ADD] l10n_es_facturae_igic

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[IMP] l10n_es_aeat_mod347: Partner record state improvements

- Track the state field for annotating in the chatter the state changes.
- Don't reset to pending the partner record state after recalculating
  if there's no change in the total amount.

TT54641

OCA-git-bot committed:
[BOT] post-merge updates

ofonseca-pyming committed:
[16.0][FIX] pinned requests_pkcs12 version due cryptography dependency

LauraCForgeFlow committed:
[IMP] l10n_es_aeat_mod390: Add field 662 to BOE export

EmilioPascual committed:
[FIX] Compute casilla 02 y 03

@moduon MT-8802

LauraCForgeFlow committed:
[FIX] l10n_es_aeat_mod390: computation of fields 97 and 662

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[FIX] l10n_es_aeat_mod303: Missing taxes in 25-26

Recargo equivalencia 0,26% and 1% for refunds are missing.

TT54712

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update l10n_es_dua_igic.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update l10n_es_facturae_igic.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

EmilioPascual committed:
[IMP] l10n_es_aeat_mod390: Casillas 658 y 84

@moduon MT-8868

pedrobaeza committed:
[IMP] l10n_es_aeat*: Show only fields with values by default

Models like 303 and 390 have a lot of fields, so showing them all even
if the amount is 0 can be counterproductive. Thanks to the magic of
one2many fields, we can continue calculating all the fields and showing
them for traceability purposes, but filter out in another o2m field
those with amount with hardly no cost, and now 2 tabs are shown for
these lines, having precedence the filtered one.

pedrobaeza committed:
[FIX] l10n_es_aeat_mod390: Missing fields 60 and 61

The 12% VAT for agriculture et al. were missing in the mapping and the
sum, so we need to include it as the tax is present and it's declared
in the 303 model.

Commit:
[UPD] Update l10n_es_aeat_mod115.pot

Commit:
[UPD] Update l10n_es_aeat_mod111.pot

Commit:
[UPD] Update l10n_es_aeat_mod123.pot

Commit:
[UPD] Update l10n_es_aeat_mod216.pot

Commit:
[UPD] Update l10n_es_aeat_mod190.pot

Commit:
[UPD] Update l10n_es_aeat_mod303.pot

Commit:
[UPD] Update l10n_es_aeat_mod296.pot

Commit:
[UPD] Update l10n_es_aeat_mod369.pot

Commit:
[UPD] Update l10n_es_aeat_mod390.pot

Commit:
[UPD] Update l10n_es_atc_mod420.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod390
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod390/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod303
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod303/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod123
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod123/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod369
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod369/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod216
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod216/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod296
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod296/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod115
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod115/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod111
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod111/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_atc_mod420
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_atc_mod420/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod190
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod190/

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update l10n_es_aeat_mod390.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod390
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod390/

OCA-git-bot committed:
[IMP] update dotfiles

Commit:
[UPD] Update l10n_es_aeat_mod390.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod390
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod390/

luisDIXMIT committed:
[IMP] l10n_es_irnr: Add xmlid rename in script migration

Commit:
[UPD] Update l10n_es_aeat_sii_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_sii_oca
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_sii_oca/

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Missing non deductible taxes

The new october taxes, in its no deductible form, were not added.

TT55046

OCA-git-bot committed:
[BOT] post-merge updates

sergiobstoj committed:
[FIX] l10n_es_aeat_mod369: Fix neg_corrections calculation, there could be mora that one refund line by country

victoralmau committed:
[IMP] l10n_es_facturae: Retención de garantía

TT55080

victoralmau committed:
[FIX] l10n_es_facturae: Activar tests + correcciones de importes

FWP parcial de https://github.com/OCA/l10n-spain/commit/62b30b88c233e1f783c4d8cb7e90e9f74be57dbe
https://github.com/OCA/l10n-spain/pull/3529#issuecomment-2058383875

Co-authored-by: ramiadavid

pedrobaeza committed:
[FIX] delivery_gls_asm: Disable tests

GLS is now returning error continuously, so we can't do real tests
against its API.

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_facturae
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_facturae/

victoralmau committed:
[IMP] l10n_es_facturae: Importe retención garantía modificable

Se define el campo facturae_withheld_amount readonly=False para permitir modificarlo
para que se pueda ajustar manualmente por ejemplo por el redondeo.

TT55080

OCA-git-bot committed:
[BOT] post-merge updates

etobella committed:
[IMP] l10n_es_facturae_face: we need to check by type, not the id of the record

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[FIX] l10n_es_aeat_mod303: New BOE format 2025

In fact, the changes are very small for the implemented things, as
the changes on the REAGP, which are the biggest, are not implemented.
We just need to force to 0 the percentages for the ephemeral taxes.
They just can ignore that fields for not bothering people...

There's also a move for "Recargo equivalencia 0.5%" from 16-18 to
168-170 inadvertently.

OCA-git-bot committed:
[BOT] post-merge updates

victoralmau committed:
[IMP] l10n_es_facturae: Restar el importe de retención de garantía del total

Relacionado con https://github.com/OCA/l10n-spain/blob/abf6a4bb0ee52305649f07036568f52c220f1c84/l10n_es_facturae/data/Facturaev3_2_2.xsd#L884

TT55080

Commit:
[IMP] l10n_es_aeat_mod347_igic: Added IGIC Exento Soportado to map

OCA-git-bot committed:
[BOT] post-merge updates

Bilbonet committed:
[FIX][16.0]l10n_es_ticketbai: fix error in post-migration script when upgrading from v15

polqubiq committed:
[FIX] l10n_es_facturae: solve traceback error when partner_id has not zip

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

EmilioPascual committed:
[FIX] l10n_es_aeat_mod190: Visible computo de los 3 primeros hijos

El computo de los 3 primeros hijos sólo debe ser visible para percepciones correspondientes a las claves A, B01, B03 y C

@moduon MT-8874

Co-authored-by: Loida Vilaplana <104626797+loida-vm@users.noreply.github.com>

rrebollo committed:
[FIX] l10n_es_ticketbai: adjust colspan for VAT exemptions field in account_fiscal_position_views

pedrobaeza committed:
[FIX] l10n_es*: Post-install test + fallback to load CoA

Since odoo/odoo@d0342c8, the default existing company is not getting a
CoA automatically, provoking than the current tests fail with errors
like:

odoo.exceptions.UserError: No journal could be found in company My Company (San Francisco) for any of those types: sale

or

psycopg2.errors.CheckViolation: new row for relation "account_move_line" violates check constraint "account_move_line_check_accountable_required_fields"

All of them provoked by the lack of a CoA installed.

Thus, we put tests post-install for being sure localization modules are installed, the same as AccountTestInvoicingCommon does, but we don't inherit from it, as it creates an overhead creating 2 new companies and loading their CoA, and some more stuff, while we don't need all of that.

Besides, if you don't have `l10n_generic_coa` installed, you can't use
another CoA (like `l10n_es`, which is here the base) easily, so we put
little code to select the first available CoA.

Another extra fixes + improvements done on the same mood.

pedrobaeza committed:
[IMP] l10n_es_location_nuts: Remove warning

WARNING odoo py.warnings: /usr/lib/python3.10/contextlib.py:135: DeprecationWarning: Since Odoo 15.0, Environment.manage() is useless.

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[FIX] l10n_es_aeat_mod303: Avoid negatives in fee to compensate

If the previous 303 report has remaining fee to compensate, and the
result of the report is positive and bigger than the fee to compensate,
when computing the current one, the constraint

  "The fee to compensate must be indicated as a positive number."

will raise always.

To avoid it, two changes have been done:

1. Only substract report result if it's of type C.
2. Nullify any possible result when it's less than 0.

Amendment of 4bd1c0280cbdaf7f452435d398fe5000ec344a6f

TT55245

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[FIX] l10n_es_atc_mod415: Added IGIC Exento Soportado to map

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[FIX] l10n_es_partner: Inject _rec_names_search properly

Previous patch was overriding _auto_init method for injecting the
comercial field in the list of fields to search for (_rec_names_search),
but this method is only called on module installation/update, so if
you restart the Odoo server after that, you lose the injection.

Thus, let's inject it if not present when calling name_search, and
this way we always have it available.

TT55241

pedrobaeza committed:
[FIX] l10n_es_aeat_sii_taxfree: Test inheritance

Inherit from TestL10nEsAeatSiiBase instead of TestL10nEsAeatSii for not
repeating tests that have been already executed.

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[FIX] l10n_es_aeat_sii_oca: Inhibit agencies test

As they are not very reliable, let's inhibit these tests for not
blocking whole repository.

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_aeat_mod190
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_aeat_mod190/

sergio-teruel committed:
[FIX] l10n_es_facturae: Constraint error when a child contact is created directly

OCA-git-bot committed:
[BOT] post-merge updates

EmilioPascual committed:
[IMP] l10n_es_aeat_mod190: Compute deductible expenses

After this commit https://github.com/odoo/odoo/pull/198546/commits/9056fc6fa21d6e9dd7088dade22075795e23eeef where odoo added tax for deductible expenses, it's possible compute deductible expenses for each recipient.

@moduon MT-8853

etobella committed:
[OU-ADD] l10n_es_facturae: Add migration Script

OCA-git-bot committed:
[BOT] post-merge updates

Anxo82 committed:
[IMP] l10n_es_facturae: remove unidad tramitadora and oficina contable checks

Anxo82 committed:
[IMP] l10n_es_facturae_face: add unidad tramitadora and oficina contable checks

etobella committed:
[OU-FIX] l10n_es_facturae: typo error

Commit:
[UPD] Update l10n_es_facturae_face.pot

Commit:
[UPD] Update l10n_es_facturae.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_facturae_face
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_facturae_face/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_facturae
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_facturae/

polcanelles committed:
[FIX] l10n_es_facturae: Align facturae_code field position with v17

In Odoo 16, the facturae_code field is misplaced because the company_id field appears twice in the form. The current inheritance rule places facturae_code after the first occurrence of company_id, which is outside the form layout and invisible.

To maintain consistency with Odoo 17, this commit repositions facturae_code after the correct company_id field inside the "Advanced Options" tab, using the same solution as in Odoo 17 to ensure proper alignment.

OCA-git-bot committed:
[BOT] post-merge updates

etobella committed:
[IMP] l10n_es_account_statement_import_n43: avoid use of retrieve_partner as we are already searching the partner

Tisho99 committed:
[IMP] l10n_es_sigaus_sale: call sigaus from sale line creation

juancarlosonate-tecnativa committed:
[FIX] l10n_es_intrastat_report: Incompatibility with other intrastat

TT55417

pedrobaeza committed:
[IMP] l10n_es_partner: Inject the field search more elegantly

Using Python @property decorator.

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

oihane committed:
[IMP] l10n_es_ticketbai_oss: pre-commit stuff

oihane committed:
[MIG] l10n_es_ticketbai_oss: Migration to 16.0

chienandalu committed:
[FIX] l10n_es_partner: display address with commercial name

Before this commit the commercial name would be display at the
end of the address. Something like this:

Paco Poco S.A.
Calle Mayor, 1
50001 Zaragoza
España (Industrias Paco)

Which is not the expected way. With this change, we can show it after the partner's name:

Paco Poco S.A (Industrias Paco)
Calle Mayor, 1
50001 Zaragoza
España

MT-8475

Commit:
[UPD] Update l10n_es_ticketbai_api.pot

Commit:
[UPD] Update l10n_es_ticketbai_oss.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_ticketbai_api
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_ticketbai_api/

OCA-git-bot committed:
[BOT] post-merge updates

pedrobaeza committed:
[FIX] l10n_es_aeat: Consider Northern Ireland as European

VAT numbers starting with XI, which comes from the EORI number
(https://www.gov.uk/eori/eori-northern-ireland), even being from
United Kingdom, are still under European intra-community operations,
so we have to include this prefix for issuing IDType=2 identifier
and accept intra-community VAT.

TT55678

chienandalu committed:
[FIX] l10n_es_partner: prevent replacing multiple times

Related to this fix #4116, let's prevent replacing the partner's
in a possible match in the address. Not very likely but possible
in any case.

MT-8475

oihane committed:
[FIX] l10n_es_ticketbai_oss: recover code removed by mistake

Commit:
[UPD] Update l10n_es_ticketbai_oss.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

herminioelx committed:
[16.0][FIX] l10n_es_facturae: Taxes withheld need to be divided by 100, just like taxes

OCA-git-bot committed:
[BOT] post-merge updates

etobella committed:
[FIX] l10n_es_facturae: Use AEAT test

OCA-git-bot committed:
[BOT] post-merge updates

herminioelx committed:
[FIX] l10n_es_facturae: The city field in the partner is required for the FacturaE process. If the user does not provide it, document generation fails and module crashes.

pedrobaeza committed:
[FIX] l10n_es_partner: Contact name may be empty

Odoo allows to have empty name on children contacts, provoking an error
when displaying the address with name_get:

  File "/opt/odoo/auto/addons/l10n_es_partner/models/res_partner.py", line 33, in _get_name
    return origin.replace(
TypeError: replace() argument 1 must be str, not bool

Let's cover that situation for avoiding the crash.

TT55861

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[BOT] post-merge updates

Tisho99 committed:
[IMP] l10n_es_sigaus modules: remove ensure_ones

Commit:
[UPD] Update l10n_es_intrastat_report.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_intrastat_report
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_intrastat_report/

pedrobaeza committed:
[FIX] l10n_es_aeat_mod347: Proper view inheritance

The locator is on an inherited view, so we have to put the inheritance
correctly.

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update l10n_es_facturae.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_facturae
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_facturae/

OCA-git-bot committed:
[BOT] post-merge updates

juancarlosonate-tecnativa committed:
[FIX] l10n_es_intrastat_report: Conflicting UserError incompatibility

TT55417

Commit:
[UPD] Update l10n_es_intrastat_report.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-spain-16.0/l10n-spain-16.0-l10n_es_intrastat_report
Translate-URL: https://translation.odoo-community.org/projects/l10n-spain-16-0/l10n-spain-16-0-l10n_es_intrastat_report/

arch-fan commented:
¿Por qué se cerró?
